draft-specifications/Model.md

```diff
@@ -236,26 +236,26 @@ Selectors are distinguished from [String](#string)s by type, not content.
 >
 > Tracking: https://github.com/ocapn/ocapn/issues/46
 >
-> OCapN uses the name Selector to avoid the implication that they will
+> Although OCapN uses the name Symbol, they will
 > correspond to a language's symbol type in all languages that have a symbol
 > type.
```

Comment:
Note that this is a meaning change to prose, not just a mechanical search-and-replace